src/System.CommandLine/RootCommand.cs

```diff
@@ -23,7 +23,17 @@ public class RootCommand : Command
                 LazyThreadSafetyMode.PublicationOnly);
 
         private static readonly Lazy<string> _executablePath =
-            new(() => GetAssembly().Location,
+            new(() =>
+                {
+#if NET6_0_OR_GREATER
+                    var al = Environment.ProcessPath ?? "";
```

Comment:
(nit) Is `al` the best variable name here?

Reply:
No, it's a terrible name! 😅 I'm still tinkering.